superset/security/manager.py

```diff
@@ -713,6 +712,7 @@ def create_custom_permissions(self) -> None:
         self.add_permission_view_menu("all_datasource_access", "all_datasource_access")
         self.add_permission_view_menu("all_database_access", "all_database_access")
         self.add_permission_view_menu("all_query_access", "all_query_access")
+        self.add_permission_view_menu("can_csv", "Superset")
```

Comment:
This is going to be a custom permissions now. I think that something like `can_download_chart` would make more sense, so I propose we do that as a fast follow.

Another alternative is to totally remove this permission, since a user that can download a CSV, XLS related with a chart already as access to that chart data.

Reply:
Would this new perm combine both CSV/XLSX etc downloading, and chart/dashboard image downloading? I think that could make sense.

Reply:
no strong opinions about it, I'll propose it